service/kvmconfig/framework.go

```diff
@@ -150,7 +150,7 @@ func NewFramework(config FrameworkConfig) (*framework.Framework, error) {
 
 			GuestUpdateEnabled: config.GuestUpdateEnabled,
 			HandledVersionBundles: []string{
-				"1.1.1",
+				"1.2.0",
```

Comment:
Issues with this happened quite often now. How can we improve this? Are there actually proper errors? Can we maybe have some metrics/alerts or even CI validation?

Reply:
I think alert when encountering unknown version bundle would be nice. This is actually user facing problem. Again we reacted quicker, than user had chance to see that.

Reply:
That we got lucky here now is basically only because there is no big demand at times. So lets not consider luck as business rationale. IMO we should have validation for this at build time. That would be best. I was thinking about connecting version bundles and assigned resources somehow but couldn't come up with something cool so far. We would gain a lot with build improvements here. Do you have a nice idea by chance?

Reply:
Yes, I think we could move handled version bundles to the kvmconfig/v* directories. Then it's enough to take all the versions and pass it to HandledVersionBundles programatically.

Reply:
I am not sure I follow. How would that solve the programmatic relation between version bundles and resources?

Reply:
Now we have that sorted! :)

Reply:
🎉